The test fails for our librosa version as ".post1" version suffix isn't accepted by `StrictVersion`

Author: Alexander Grund (TU Dresden)

diff -ur b/test/torchaudio_unittest/functional/librosa_compatibility_test_impl.py a/test/torchaudio_unittest/functional/librosa_compatibility_test_impl.py
--- b/test/torchaudio_unittest/functional/librosa_compatibility_test_impl.py	2026-01-14 08:53:30.541831678 +0100
+++ a/test/torchaudio_unittest/functional/librosa_compatibility_test_impl.py	2026-01-14 08:53:47.172772868 +0100
@@ -77,8 +77,6 @@
     def test_create_mel_fb(
         self, n_mels=40, sample_rate=22050, n_fft=2048, fmin=0.0, fmax=8000.0, norm=None, mel_scale="htk"
     ):
-        if norm == "slaney" and StrictVersion(librosa.__version__) < StrictVersion("0.7.2"):
-            self.skipTest("Test is known to fail with older versions of librosa.")
         if self.device != "cpu":
             self.skipTest("No need to run this test on CUDA")
 
